src/dcos_e2e/_vendor/dcos_launch/onprem.py

```diff
@@ -109,7 +109,7 @@ def get_completed_onprem_config(self) -> typing.Tuple[dict, str]:
             # use a sensible default
             shutil.copyfile(
                 pkg_resources.resource_filename(
-                    'dcos_launch', script_hyphen + '/{}.sh'.format(self.config['platform'])),
+                    'dcos_e2e._vendor.dcos_launch', script_hyphen + '/{}.sh'.format(self.config['platform'])),
```

Comment:
We should document this manual override in `contributing.rst` and link to a JIRA for upstreaming it.

Reply:
(Tim did this)